packages/mux-player/test/web-test-runner.config.mjs

```diff
@@ -1,8 +1,20 @@
 import { esbuildPlugin } from '@web/dev-server-esbuild';
+import { importMapsPlugin } from '@web/dev-server-import-maps';
 
 export default {
   nodeResolve: true,
   plugins: [
+    importMapsPlugin({
+      inject: {
+        importMap: {
+          imports: {
+            '/test/': '/packages/mux-player/test/',
+            'hls.js': '/node_modules/@mux-elements/test-esm-exports/dist/hls.js',
+            'mux-embed': '/node_modules/@mux-elements/test-esm-exports/dist/mux-embed.js',
```

Comment:
nice! I was kind of bummed out because of the need to build all element pkgs before running test.

Reply:
It should be possible to use this to configure the modules for testing to point at the TS code. Though, I think if we wanted to go that route, we should do it in a separate PR.
`'mux-audio': '/node_modules/@mux-elements/mux-audio/src/index.ts'`
or even directly at the package with `'/packages/mux-audio/src/index.ts'` and since web-test-runner runs things through esbuild, it should work.